DualView/app/src/main/java/com/microsoft/device/display/samples/dualview/RestaurantView.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *  *
+ *
+ */
+
+package com.microsoft.device.display.samples.dualview
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.preferredHeight
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import androidx.navigation.compose.navigate
+import com.microsoft.device.display.samples.contentcontext.util.formatPriceRange
+import com.microsoft.device.display.samples.contentcontext.util.formatRating
+import com.microsoft.device.display.samples.dualview.models.AppStateViewModel
+import com.microsoft.device.display.samples.dualview.models.Restaurant
+import com.microsoft.device.display.samples.dualview.models.restaurants
+import com.microsoft.device.display.samples.dualview.ui.theme.selectedBody1
+import com.microsoft.device.display.samples.dualview.ui.theme.selectedBody2
+import com.microsoft.device.display.samples.dualview.ui.theme.typography
+
+private val outlinePadding = 25.dp
```

Comment:
is this using an extension function to convert it to dps?

Reply:
No, why do we need a extension function?

Reply:
No, I mean, is `.dp` an extension function?

Reply:
No, it is just how the unit is used in Jetpack Compose. https://developer.android.com/reference/kotlin/androidx/compose/ui/unit/package-summary#dp_2

Reply:
Ah oki, so they are extension functions created for and in the compose framework. Cool!